README.md

```diff
@@ -70,9 +71,9 @@ npm run html-validator
 
 ## 👤 Author
 
-- Github: [@author](https://github.com/author)
-- Twitter: [@author](https://twitter.com/author)
-- Linkedin: [@author](https://www.linkedin.com/in/author/)
+- Github: [@author](https://github.com/bimbolabuari)
+- Twitter: [@author](https://twitter.com/bimbolabuari)
+- Linkedin: [@author](https://www.linkedin.com/in/bimbolabuari)
```

Comment:
I think we can remove the `@author` and add your handle, e.g. `@bimbolabuari`.

Also, let's replace `## 👤 Author` with your name `## 👤 YOUR NAME HERE`